test/performance/odp_pktio_ordered.c

```diff
@@ -1025,7 +1046,10 @@ static void print_info(char *progname, appl_args_t *appl_args)
 	       progname, appl_args->if_count);
 	for (i = 0; i < appl_args->if_count; ++i)
 		printf(" %s", appl_args->if_names[i]);
-	printf("\n\n");
+	printf("\n");
+	printf("Promisc mode:    %s\n", appl_args->promisc_mode ?
```

Comment:
Currently the application parameters are printed in two places, here and in `main():1291`. You could move also the rest of the prints here, so everything would be printed nicely in one place.

Reply:
Moved.